package.json

```diff
@@ -1,28 +1,40 @@
 {
-  "name": "electron-sentry",
-  "description": "JavaScript and native crash reporting from Electron to Sentry",
-  "version": "1.1.2",
+  "name": "@sentry/electron",
+  "description": "Offical Sentry SDK for Electron",
+  "version": "0.1.0",
```

Comment:
This will need to be greater than the last to publish to npm.

As the entire API has changed v2 would be preferable!

Reply:
Or do you guys want to just publish to `@sentry/electron`?

Reply:
That would be the idea. We could experiment a little more with the API before making it stable then